src/main/java/Task.java

```diff
@@ -0,0 +1,15 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "true" : "false"); //return tick or X symbols
```

Comment:
Coding Standard: can put comment one line above with the same indentation as the return statement